src/main/java/gov/usgs/wma/waterdata/parameter/DataType.java

```diff
@@ -0,0 +1,10 @@
+package gov.usgs.wma.waterdata.parameter;
+
+public enum DataType {
+	any,
+	statistical_time_series;
+	
+	public boolean isStatisticalTimeSeries() {
+		return this.equals(any) || this.equals(statistical_time_series);
+	}
+}

```

Comment:
I don't understand this enum.  Typical enum conventions would be to have all caps values.

Reply:
The enum is lowercase, to allow for the value in the query parameter to be lowercase. Since spring is directly
mapping it at the moment. This should be updated to a case insensitive mapping for the query parameter
and the enum values changed to upper case.